File: crates/codegraph-python/tests/fixtures/simple.py
"""Simple Python module for testing basic parsing."""


def greet(name: str) -> str:
    """Greet someone by name."""
    return f"Hello, {name}!"


class Person:
    """A simple person class."""
    
    def __init__(self, name: str, age: int):
        self.name = name
        self.age = age
    
    def introduce(self) -> str:
        """Return introduction string."""
        return f"My name is {self.name} and I am {self.age} years old."

File: crates/codegraph-swift/tests/fixtures/simple.swift
// Simple Swift file for testing

import Foundation

/// A simple class for testing
class Person {
    var name: String
    var age: Int

    init(name: String, age: Int) {
        self.name = name
        self.age = age
    }

    func greet() -> String {
        return "Hello, \(name)!"
    }
}

/// A free function
func greetWorld() -> String {
    return "Hello, World!"
}
